### ISSUE TITLE
nvda doesn't read anything when you re-open the latest versions of skype

### ISSUE THREAD

**Issue by Nikita34196** [nvaccess/nvda#11879]
Hello. I've had this problem for months. This happens if skype is running in the background. this problem also occurs when skype is re-opened. 1. Try opening the skype after turning on the pc after a few minutes. you will see that the main window is not readable. you can only read with ocr. it also happens if skype is not in the auto-start, but you started it held a conversation and it was completely closed. when the nvda is reopened, the main window stops reading.
windows 10 20262 NVDA version alpha-21429,874230a7. nvda installed. skype 8.67.76.64

**Comment by k-kolev1985** [comment]
And this problem is not fault of NVDA. It is a problem in Skype, so should be reported to the Skype development team. A workaround when NVDA stops speaking in Skype is to press ALT to go to the menu bar and then press ESCAPE to get out of the menu bar - that will temporarily fix the issue, until the window of Skype is closed.

**Comment by Adriani90** [comment]
@Nikita34196, @k-kolev1985 I cannot reproduce this with NVDA 2023.1 RC1 and Skype 94. Are you still having this issue with these versions?

**Comment by Adriani90** [comment]
Thanks for the quick update. Closing as works for me.
